integration_tests/cleanup-jest-html-reports.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Set the number of directories to keep
+num_of_dir_to_keep=$1
+
+# Change to the target directory
+cd "$2"
+
```

Comment:
You can leave this out completely and do the `cd` outside of the script (as I show in the github-action).

Reply:
removed this and created npm script with cd